README.md

```diff
@@ -33,6 +33,6 @@ https://revspace.nl/MHZ19
 ## Further reading:
 - http://ihormelnyk.com/Page/mh-z19_co2_meter
 
-(russuian, but google translate does a good job)
-- https://geektimes.ru/post/285572/
-- https://geektimes.ru/post/278178/
+Russian, but google translate does a good job:
+- [https://geektimes.ru/post/285572/](https://habr-com.translate.goog/en/post/401363/?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=nl)
```

Comment:
thanks for the contribution, but I think urls should lead to where they say
also for most users the browser will already offer translation

Reply:
Hi, you’re right.
Actually I only wanted to contribute a fix for the typo in “russuian”.
Kind regards